kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/ProxyDeployment.java

```diff
@@ -19,10 +19,12 @@
 
 import io.kroxylicious.kubernetes.api.v1alpha1.KafkaProxy;
 
+import static io.kroxylicious.kubernetes.operator.StandardLabels.addStandardLabels;
+
 /**
  * The Kube {@code Deployment} for the proxy
  */
-@KubernetesDependent
+@KubernetesDependent(labelSelector = "app.kubernetes.io/managed-by=kroxylicious-operator")
```

Comment:
I'm not sure we want to do this. What ought to be the behaviour of the Deployment with the required name exists but without the annotations on? I would argue it should be operated on anyway (and thus the labels added). Also I think these labels are really intended to be informational (so someone can look at the Deployment, and figure out what's managing it)m.

Reply:
Okay, so it could then recover from users or tools manipulating labels directly.

Reply:
But if the operator acts on it regardless of the annotation we need to do something else to have the CR exist but not managed by the operator.